DeepMRSeg/deepmrseg_test.py

```diff
@@ -8,8 +8,8 @@
 #- Make sure the output datatypes are efficient and readable by 3dcalc/fslmaths
 
 ################################################ DECLARATIONS ################################################
-__author__ 	= 'Jimit Doshi'
-__EXEC_NAME__ 	= "tf_segunet_test"
```

Comment:
It seems all the tabs have been replaced by 4/8 spaces. Since all the other files use tabs, can we switch back to using tabs for consistency?

Reply:
done..